src/scripts/OSFramework/OSUI/Pattern/DropdownServerSideItem/scss/_dropdownserversideitem.scss

```diff
@@ -17,10 +17,12 @@
 	position: relative;
 	transition: background 250ms ease;
 	width: 100%;
+	z-index: 0;
```

Comment:
One question, we don't need to use the layer systems here? Why we use the zindex 0 and 1?

Reply:
This is the same element...